0-content-page/FCCvote-net-neutrality-share/page-content.html

```diff
@@ -0,0 +1,66 @@
+<!-- =================== Page Content STARTS =================== -->
+
+<div id="wrapper">
+  <header id="masthead">
+    <div class="container">
+      <a href="https://www.mozilla.org/" id="tabzilla">mozilla</a>
+      <a href="https://www.mozilla.org" id="header-logo">
+        <img src="https://sendto.mozilla.org/page/-/protect-net-neutrality/img/mozilla-wordmark.png" alt="Mozilla Wordmark" class="mozilla-wordmark">
+      </a>
+    </div>
+  </header>
+  <main>
+    <div class="container">
+      <div id="header">
+        <h1>Thanks for signing up</h1>
+        <h3>Will you do one more thing and share this action with your friends?</h3>
+      </div>
+      <div id="white-box">
+        <div>
+          <h2>Share</h2>
+          <div class="row">
+            <div class="half">
+              <a href="http://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fmzl.la%2F169Y0wn" id="facebook-share" class="share-btn">
+                <i class="fa fa-facebook fa-5x"></i>
+                <p>facebook</p>
+              </a>
+            </div>
+            <div class="half">
+              <a href="http://twitter.com/share?text=I%20just%20took%20action%20to%20protect%20net%20neutrality.%20The%20FCC%20votes%20soon!%20Take%20action%20here:&url=http%3A%2F%2Fmzl.la%2F1K55aRm" id="twitter-share" class="share-btn">
+                <i class="fa fa-twitter fa-5x"></i>
+                <p>twitter</p>
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </main>
+  <footer>
+    <div class="container">
+      <div id="wordmark-and-note">
+        <a href="https://www.mozilla.org">
+          <img src="https://sendto.mozilla.org/page/-/protect-net-neutrality/img/mozilla-wordmark.png" alt="Mozilla Wordmark" class="mozilla-wordmark">
+        </a>
+        <p class="license">
+          Portions of this content are &copy;1998&ndash;2015 by individual mozilla.org contributors. Content available under a <a href="https://www.mozilla.org/foundation/licensing/website-content/" target=\"_blank\">Creative Commons license</a>.
+        </p>
+      </div>
+      <ul class="links">
+        <li><a href="https://www.mozilla.org/mission/" target=\"_blank\">Mission</a></li>
+        <li><a href="https://www.mozilla.org/about/" target=\"_blank\">About</a></li>
+        <li><a href="https://www.mozilla.org/contact/spaces/" target=\"_blank\">Contact</a>
+        </li>
+        <li><a href="https://www.mozilla.org/privacy/" target=\"_blank\">Privacy Policy</a>
+        </li>
+        <li><a href="https://www.mozilla.org/about/legal/" target=\"_blank\">Legal Notices</a>
+        </li>
+        <li><a href="https://sendto.mozilla.org/page/contribute/givenow-seq?preset=2&ref=Advocacy_Website&utm_campaign=Advocacy_Website&utm_source=advocacy.mozilla.org&utm_medium=referral" target=\"_blank\" class="red">Donate</a>
```

Comment:
@gesa I replaces all the `&` with `%26` and now clicking on that link brings me to `https://sendto.mozilla.org/page/contribute/givenow-seq?preset=2%26ref=Advocacy_Website%26utm_campaign=Advocacy_Website%26utm_source=advocacy.mozilla.org%26utm_medium=referral#page-1` I imagine the `%26` will be auto converted back to `&`... am I wrong? More curious about if that long link above will still be considered in the correct query param format (ie, `%26` instead of `&`)